modules/ROOT/pages/producing-ign.adoc

```diff
@@ -0,0 +1,60 @@
+:experimental:
+= Fedora CoreOS - Producing an Ignition File
+
+This guide provides easy, step-by-step instructions to produce a basic Ignition file.
+
+Fedora CoreOS (FCOS) Ignition files specify the configuration of FCOS. The process begins with a YAML configuration file. The FCOS Configuration Transpiler (FCCT) converts the human-friendly YAML file into machine-friendly JSON.
+
+TIP: Once FCOS boots and ingests the Ignition file, the configuration is meant to be immutable. Therefore, plan your Ignition file with the full configuration specifications that you will need.
+
+== A simple example
+Create a basic Ignition file that modifies the default FCOS user `core` to allow this user login with an SSH key.
+
+The overall steps are as follows:
+
+. Write the Fedora CoreOS Configuration (FCC) file in the YAML format.
+. Download and install the Fedora CoreOS Configuration Transpiler (`fcct`).
+. Use `fcct` to convert the FCC file into an Ignition (JSON) file.
+
+=== Prerequisite
+Create the `core` user on your local machine, and https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/sec-security#sec-SSH[generate an SSH key] for this user. When the user logs in via SSH, you will use the SSH key from the local host.
+
+=== Writing the FCC file
+
+. Copy the following example into a text editor:
++
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - ssh-rsa AAAA...
+----
++
+. Replace the above line starting from `ssh` with the contents of core's SSH public key file.
++
+. Save the file with the name `example.fcc`.
+
+TIP: YAML files must have consistent indentation. Although `fcct` checks for syntax errors, ensure that the indentation matches the above example. Overall, the FCC files must conform to ``fcct``'s xref:fcct-config.adoc[schema].
+
+=== Using FCCT
```

Comment:
Hmm, this section seems redundant with `using-fcct.adoc` -- we can just link to that doc it seems?